configure.ac

```diff
@@ -2774,6 +2774,10 @@ on all link lines (consider adding it to LDFLAGS)])
     ## most Unix versions, libtvmpich.dylib for Mac OSX).
     ##ENVVAR END:
 
+    if test "$device_name" = "ch4" ; then
+        AC_DEFINE(HAVE_CH4_DEBUGGER_SUPPORT,1,[Define if debugger support is included for CH4])
```

Comment:
I think we need a comment here explaining why this is needed.

Reply:
Another hack solution we used in the hcoll integration code was to check if `MPIDCH4_H_INCLUDED` is defined. Maybe that could work and we could drop this? It's still hacky, but I don't have a solution for that right now.

Reply:
@raffenet Currently, `MPIDCH4_H_INCLUDED` is not working under `/home/caochong/git/mpich-ofi/src/mpi/debugger`. It is only useful under `mpid`.

Reply:
I see. That's true of `dll_mpich.c` since it only includes `mpi.h` for some reason. The files that include `mpiimpl.h` should be able to use `MPIDCH4_H`, but as I mentioned, it is super hacky.

Reply:
I have added some comments.